types/gl/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for gl 4.1
+// Project: https://github.com/stackgl/headless-gl#readme
+// Definitions by: sjx233 <https://github.com/sjx233>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare global {
+    interface WebGLRenderingContext {
+        getExtension(extensionName: "STACKGL_resize_drawingbuffer"): STACKGL_resize_drawingbuffer | null;
+        getExtension(extensionName: "STACKGL_destroy_context"): STACKGL_destroy_context | null;
+    }
+}
+
+export interface STACKGL_resize_drawingbuffer {
+    resize(width: GLint, height: GLint): void;
+}
+
+export interface STACKGL_destroy_context {
+    destroy(): void;
+}
+
+export default function createContext(width: number, height: number, options?: WebGLContextAttributes): WebGLRenderingContext;
```

Comment:
This needs export = instead of default. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

Reply:
5aa0be6